src/components/Nav/Search/Search.js

```diff
@@ -69,36 +61,23 @@ class Search extends Component {
 
   searchMovie = (event) => {
     event.preventDefault();
-    const { searchValue } = this.state;
-    const { searchData } = this.props;
-    let searchPool = [];
-    if (searchData) {
-      searchPool = searchData.data;
-    }
+    const { searchValue, searchData } = this.state;
     const searchKeywords = searchValue.split(' ');
-    let tempSearchPool = [...searchPool];
-    let tempFilteredMovie = [];
-    if (searchValue.trim() === '') {
-      return;
-    }
-    searchKeywords.forEach((key) => {
-      if (key === '') return;
-      tempFilteredMovie = tempSearchPool.filter((movie) => {
-        if (movie.title.includes(key)) {
-          return true;
-        }
+    const [...filteredMovie] =
+      searchValue &&
+      [...searchData].filter((movie) => {
+        return searchKeywords.every((keyword) => movie.title.includes(keyword));
```

Comment:
```suggestion
    const filteredMovie = searchValue &&
      [...searchData].filter((movie) => {
        return searchKeywords.every((keyword) => movie.title.includes(keyword));
```
변수 선언하는 곳에 스프레드 연산자 쓰더라도 의미가 없습니다

Reply:
아하..... 그러면... 스프레드 연산자 빼면 스테이트에 직접 접근하는 것이니까 새롭게 tempFiltered 리스트 선언하여 변경했습니다!